printer_mods/warriorofwire/openbuilds_drag_chain_2.2/README.md

```diff
@@ -0,0 +1,28 @@
+# Off The Shelf Drag Chains for 2.2
+
+![Render](https://user-images.githubusercontent.com/3454741/68000301-c66fa480-fc1c-11e9-9096-63312416205e.png)
```

Comment:
Add the image to the PR and use relative linking instead of deep links to other images somewhere on github which might be moved/disappear at some point